docs/developer/procedures/release.rst

```diff
@@ -152,17 +152,19 @@ uploaded to the following location::
 Test the Artifacts
 ------------------
 
+
 Download and try out some of the artifacts from the above location and do a 
 quick smoke test that there are no issues. Engage other developers to help 
 test on the developer list.
 
-In particular, you can download the source artifacts and build them locally on an empty Maven repository to make sure any random user out there can do the same.
-
-A simple way to do so is:
+Check the artifacts by:
+*  Unpacking the sources
+*  Checking the README.html links go to the correct stable or maintenance user guide
 
-*  Unpack the sources
-*  Check the README.html links go to the correct stable or maintenance user guide
-*  Temporarily move the ``$HOME/.m2/repository`` to a different location, so that Maven will be forced to build from an empty repo. If you don't want to fiddle with your main repo just use ``mvn -Dmaven.repo.local=/tmp/m2 install -Dall -T1C`` where it points to any empty directory.
+The Jenkins job will perform a build of the source artifacts on an empty Maven
+repository to make sure any random user out there can do the same. If you want
+you still manually test the artifacts by:
```

Comment:
Small typo here - Should be "you _can_ still manually test the artifacts"

Reply:
Thank you @tbarsballe 
PR updated